admin/choose_project.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+$stmt = $db->query("SELECT `id`,`title`FROM `projects`;");
+$projects_data = $stmt->fetchAll();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Choose project</title>
+</head>
+<body>
+
+    <form method="GET" action="edit_project.php">
```

Comment:
Nothing wrong with using method GET but your other forms are POST.  I would consider making this consistent i.e using either GET or POST

Reply:
Good idea. I used GET in this instance so that the project id would appear in the URL making it obvious which project I am editing.

Reply:
i agree with patrick on this one. Get means you could share the url to edit a specific project, which makes sense